okhttp/src/main/java/io/grpc/okhttp/OkHttpClientTransport.java

```diff
@@ -416,6 +501,26 @@ public OkHttpClientStream newStream(
     }
   }
 
+  private Optional<TrustManager> getX509ExtendedTrustManager(TlsChannelCredentials tlsCreds)
+      throws GeneralSecurityException {
+    TrustManager[] tm = null;
+    // Using the same way of creating TrustManager from
+    // {@link OkHttpChannelBuilder#sslSocketFactoryFrom}.
+    if (tlsCreds.getTrustManagers() != null) {
+      tm = tlsCreds.getTrustManagers().toArray(new TrustManager[0]);
+    } else if (tlsCreds.getRootCertificates() != null) {
+      tm = io.grpc.internal.CertificateUtils.createTrustManager(tlsCreds.getRootCertificates());
```

Comment:
Import `CertificateUtils`? Is there a name collision I'm not seeing?

Reply:
Done.